module/VuFindTheme/tests/unit-tests/src/VuFindTest/ThemeResourceContainerTest.php

```diff
@@ -80,10 +80,24 @@ public function testLess()
     public function testJs()
     {
         $container = new ResourceContainer();
-        $container->addJs(['a', 'b', 'c']);
-        $container->addJs('c');
-        $container->addJs('d');
-        $this->assertEquals([], array_diff(['a', 'b', 'c', 'd'], $container->getJs()));
+        $container->addJs('a');
+        $container->addJs(['b', 'c']);
+        $container->addJs(['file' => 'd']);
+        $container->addJs('a');
+        $container->addJs(['file' => 'a']);
+        $container->addJs(['file' => 'd', 'position' => 'header']);
+        $container->addJs('http://foo/bar:lt IE 7');
+        $container->addJs([]);
+
+        $expectedResult = [['file' => 'a', 'position' => 'header'],
+                           ['file' => 'b', 'position' => 'header'],
+                           ['file' => 'c', 'position' => 'header'],
+                           ['file' => 'd', 'position' => 'header'],
+                           ['file' => 'http://foo/bar',
+                            'position' => 'header',
+                            'attributes' => ['conditional' => 'lt IE 7']],
+                        ];
+        $this->assertEquals($container->getJs(), $expectedResult);
```

Comment:
Minor point, but you should reverse the order of these parameters: in assertions, the expected value comes before the actual value (otherwise, error messages may come out in a misleading way).